Scripts/Services/SpeechToText/V1/SpeechToTextServiceExtension.cs

```diff
@@ -539,10 +533,10 @@ private void OnListenMessage(WSConnector.Message msg)
             if (msg is WSConnector.TextMessage)
             {
                 WSConnector.TextMessage tm = (WSConnector.TextMessage)msg;
-                Dictionary<string, object> customData = new Dictionary<string, object>();
-                customData.Add(Constants.String.JSON, tm.Text);
-                if (tm.Headers != null && tm.Headers.Count > 0)
-                    customData.Add(Constants.String.RESPONSE_HEADERS, tm.Headers);
+                //Dictionary<string, object> customData = new Dictionary<string, object>();
```

Comment:
Shouldn't we just remove this part instead of commenting it out.

Reply:
ok I removed the block